src/lib/testFlags.ts

```diff
@@ -47,6 +47,10 @@ class TestFlags {
   get displayPrivyLogin() {
     return !!this.queryParams.displayprivylogin || true;
   }
+
+  get enableComplianceApi() {
+    return !!this.queryParams.complianceapi;
```

Comment:
👍 side note, not your change, but I'm confused by line 48 - isn't that always true (or am I just bad at reading code)